scripts/reranker_ibm_colbert.py

```diff
@@ -70,23 +69,28 @@ def get_topics_from_qrun(path: str) -> Set[str]:
             res.add(line.split()[0])
     return sort_str_topics_list(res)
 
+
 def sort_str_topics_list(topics: List[str]) -> List[str]:
     res = sorted([int(t) for t in topics])
     return [str(t) for t in res]
 
 
-def evaluate(qrels_path: str, run_path: str, options: str = ''):        
+def evaluate(qrels_path: str, run_path: str, options: str = ''):
     curdir = os.getcwd()
     if curdir.endswith('scripts'):
-       anserini_root = '../../anserini'
+        anserini_root = '../../anserini'
     else:
-       anserini_root = '../anserini'
-    prefix = f"{anserini_root}/tools/eval/trec_eval.9.0.4/trec_eval -c -M1000 -m all_trec {qrels_path}"
+        anserini_root = '../anserini'
+    prefix = f"{anserini_root}/tools/eval/trec_eval.9.0.4/trec_eval \
```

Comment:
Can we break this dependency on anserini for eval? `trec_eval` exists... we can do something like `python -m pyserini.eval.trec_eval ...`

Reply:
The eval step is updated now.